ddtrace/tracer/textmap_test.go

```diff
@@ -216,8 +217,7 @@ func TestB3(t *testing.T) {
 
 		assert := assert.New(t)
 		assert.Nil(err)
-
-		assert.Equal(headers[b3TraceIDHeader], strconv.FormatUint(root.TraceID, 16))
+		assert.Equal(headers[b3TraceIDHeader], fmt.Sprintf("%016x", strconv.FormatUint(root.TraceID, 16)))
```

Comment:
Thanks for the help, Kyle! Upon further inspection, I think `fmt.Sprintf("%016s"` would be correct for adding the leading zeros and not altering the value of the IDs.

Reply:
I think it would be better to use *just* `Sprintf` and leave out the `FormatUint` altogether. 
My understanding is that `Sprintf("%016x", root.TraceID)` should be sufficient for formating a 16-character hex string.

Reply:
Ah, understood. Will add to both ids

Reply:
Though I think the problem I ran into with removing `FormatUint` was a type mismatch (as the id is `uint64`)